app/src/main/java/com/hms/referenceapp/photoapp/data/repository/CloudDbRepository.kt

```diff
@@ -109,7 +108,6 @@ class CloudDbRepository @Inject constructor(
         cloudDB.createObjectType(ObjectTypeInfoHelper.getObjectTypeInfo())
     }
 
-    @ExperimentalCoroutinesApi
```

Comment:
Does this change effect project in some conditions? If we are sure about it than it's fine to remove this

Reply:
When I tested the relevant parts, I didn't have a problem and I saw such an expression in Stackoverflow, I don't think we need to use it anymore. "UPDATE: Kotlin coroutines are no longer experimental as of Kotlin 1.3."

Reply:
I got it, thanks!!